internal/apiserver/route_admin_put_config_record.go

```diff
@@ -60,7 +37,6 @@ var putConfigRecord = &oapispec.Route{
 	JSONInputValue:  func() interface{} { return fftypes.JSONAnyPtr("{}") },
```

Comment:
So this will now not have a well-defined JSON schema in OpenAPI?

Reply:
We should probably get rid of this one

Reply:
The input schema for this one is any key/value mapping, so I think this is correct